integration-cli/docker_cli_help_test.go

```diff
@@ -249,7 +249,8 @@ func testCommand(cmd string, newEnvs []string, scanForHome bool, home string) er
 		// If a line starts with 4 spaces then assume someone
 		// added a multi-line description for an option and we need
 		// to flag it
-		if strings.HasPrefix(line, "    ") {
+		if strings.HasPrefix(line, "    ") &&
+			!strings.HasPrefix(strings.TrimLeft(line, " "), "--") {
```

Comment:
As it says in the comment, this test assumes that any place there are four consecutive spaces it is due to a multi-line description.

However this assumption is no longer valid.  Any flag that only has a long version flag will start with more spaces, because cobra aligns the short flags and long flags into columns.

The test was updated to reflect this output. If the first characters on the line (after the whitespace) are `--` then we know this is not a multi-line description, it's just a flag that only has a long version.